xmtp_mls/src/groups/message_history.rs

```diff
@@ -797,6 +884,22 @@ mod tests {
         .await;
 
         _m.assert_async().await;
+        std::fs::remove_file(output_path).expect("Unable to remove test output file");
+    }
+
+    #[tokio::test]
+    async fn test_prepare_history_reply() {
+        let wallet = generate_local_wallet();
+        let amal_a = ClientBuilder::new_test_client(&wallet).await;
+        let amal_b = ClientBuilder::new_test_client(&wallet).await;
+        assert_ok!(amal_b.allow_history_sync().await);
+
+        amal_a.sync_welcomes().await.expect("sync_welcomes");
+
+        let request_id = new_request_id();
+        let url = "http://0.0.0.0:5558/upload";
```

Comment:
ignore this for now.  I was testing with a locally running `message-history-server`; I'll fix this test in a minute..

Reply:
the upload server is now mocked in this test.